test/jsonapi/plugs/transform_parameters_test.exs

```diff
@@ -0,0 +1,69 @@
+defmodule DailyBugleWeb.TransformParametersTest do
```

Comment:
I think this namespace is a mistake 😁 

Shall we add doctest to this as well since we have it setup in the module?

Reply:
> I think this namespace is a mistake

D'oh! Wanna guess what the code name of my side project is? ;)

> Shall we add doctest to this as well since we have it setup in the module?

Yeah, may as well!

Reply:
I tried to set up some doc tests, but it was quite awkward.